docs/examples.md

```diff
@@ -157,6 +157,7 @@ check_task:
   check_script: gradle check
   cleanup_before_cache_script:
     - rm -rf ~/.gradle/caches/$GRADLE_VERSION/
+    - rm -rf ~/.gradle/caches/transforms-1 ~/.gradle/caches/journal-1
```

Comment:
We should either combine `$GRADLE_VERSION`, `transforms-1` and `journal-1` in one-line command or have three different lines for consistency.